docs/protocols/CTMeta.md

```diff
@@ -14,29 +14,29 @@ CKB Transaction Metadata (aka. CTMeta) is the standard of additional metadata fo
 
 CTMeta uses `group_witnesses[0].output_type` as the data container. The data should be written in Json format, with the following schema.
 
-```json
+```yaml
```

Comment:
Why change the data type ? It is indeed data in json format.

Reply:
Yep, indeed the data is JSON, but comments are **not** (as JSON doesn't support comments). On the other side YAML is fully backward compatible with JSON and supports comments!